### DISCUSSION TITLE
[BUG] Change url option not work with admin url

### DISCUSSION THREAD

**Discussion by kytosai** [strapi/strapi#6459] [Legacy Topics]
**Describe the bug**

- I setup a container with the domain (http://dev-kytosai.com) edited in the hosts file

- I try setup url with document - https://strapi.io/documentation/v3.x/deployment/nginx-proxy.html#strapi-server 
  - I change url to http://dev-kytosai.com/api

```javascript
module.exports = ({ env }) => ({
  host: env('HOST', '0.0.0.0'),
  port: env.int('PORT', 1337),
  url: 'http://dev-kytosai.com/api',
});
```

- Page http://dev-kytosai.com work (demo: https://prnt.sc/sqp9w4), but when i click to `Create the first administrator` open a new tab to http://dev-kytosai.com/api/admin

- Page have wrong static url, page don't work - Demo: https://prnt.sc/sqpan1 
- Log from container - https://prnt.sc/sqpgtl 

**System**

- Node.js version: v12
- NPM version: 
- Strapi version: v3.0.1
- Database: mongo
- Operating system: Docker container on MacOS

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
